archinstall/lib/installer.py

```diff
@@ -92,8 +92,6 @@ def __init__(
 
 		self.post_base_install: list[Callable] = []  # type: ignore[type-arg]
 
-		# TODO: Figure out which one of these two we'll use.. But currently we're mixing them..
-		storage['session'] = self
```

Comment:
This might be a breaking change for those scripting their own installations (not everyone uses `guided.py` or what's referenced in this repo.

This is mainly a note to self so I don't forget to mention that in the release notes.
We should perhaps have gone out with a breaking change notification in the previous release, but it's an easy `sed -i "s/storage['session']/storage['installation_session']/g' <file>` wherever it's used.